src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -234,14 +221,23 @@ export class MdAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
     }
   }
 
+  _handleFocus(): void {
+    this._attachOverlay();
+    this._floatPlaceholder(true);
+  }
+
   /**
    * In "auto" mode, the placeholder will animate down as soon as focus is lost.
    * This causes the value to jump when selecting an option with the mouse.
    * This method manually floats the placeholder until the panel can be closed.
+   * @param shouldAnimate Whether the placeholder should be animated when it is floated.
    */
-  private _floatPlaceholder(): void {
+  private _floatPlaceholder(shouldAnimate = false): void {
     if (this._inputContainer && this._inputContainer.floatPlaceholder === 'auto') {
-      this._inputContainer.floatPlaceholder = 'always';
+      if (!shouldAnimate) {
+        this._inputContainer.floatPlaceholder = 'always';
```

Comment:
The reason why we set the placeholder to 'always' in this fashion is to avoid the placeholder jumping when an option is selected by the mouse (when previously no option was selected). Testing it locally, it looks like this change causes the placeholder to jump again. Can you take a look?